
package publicador;

import jakarta.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 4.0.5
 * 2024-11-19T15:29:24.877-03:00
 * Generated source version: 4.0.5
 */

@WebFault(name = "UsuarioRepetidoException", targetNamespace = "http://publicador/")
public class UsuarioRepetidoException_Exception extends Exception {

    private publicador.UsuarioRepetidoException faultInfo;

    public UsuarioRepetidoException_Exception() {
        super();
    }

    public UsuarioRepetidoException_Exception(String message) {
        super(message);
    }

    public UsuarioRepetidoException_Exception(String message, java.lang.Throwable cause) {
        super(message, cause);
    }

    public UsuarioRepetidoException_Exception(String message, publicador.UsuarioRepetidoException usuarioRepetidoException) {
        super(message);
        this.faultInfo = usuarioRepetidoException;
    }

    public UsuarioRepetidoException_Exception(String message, publicador.UsuarioRepetidoException usuarioRepetidoException, java.lang.Throwable cause) {
        super(message, cause);
        this.faultInfo = usuarioRepetidoException;
    }

    public publicador.UsuarioRepetidoException getFaultInfo() {
        return this.faultInfo;
    }
}
